### ISSUE TITLE
vehicle_positions not being filled in the DB?

### ISSUE THREAD

**Issue by nukeador** [BlinkTagInc/node-gtfs#167]
Since I've upgraded to 4.14.1 and changed to the new config structure, vehiclepositions are being read but the vehicle_positions table on the sqlite is always empty

> Downloading GTFS-Realtime from http://212.170.201.204:50080/GTFSRTapi/api/vehicleposition
> Importing - 80 entries imported

```
const config = {
  sqlitePath: `${GTFS_DIR}/database.sqlite`,
  agencies: [
    {
      agency_key: 'AUVASA',
      path: `${GTFS_DIR}/static`,
      "realtimeAlerts": {
        "url": "http://212.170.201.204:50080/GTFSRTapi/api/alert",
      },
      "realtimeTripUpdates": {
        "url": "http://212.170.201.204:50080/GTFSRTapi/api/tripupdate",
      },
      "realtimeVehiclePositions": {
        "url": "http://212.170.201.204:50080/GTFSRTapi/api/vehicleposition",
      }
    },
  ],
  ignoreDuplicates: true,
};
```

**Comment by brendannee** [comment]
Great catch - this is now fixed in version 4.14.2 https://github.com/BlinkTagInc/node-gtfs/releases/tag/4.14.2

**Comment by nukeador** [comment]
Fix confirmed, thanks!
